translib/transformer/cfg_mgmt_xfmr.go

```diff
@@ -0,0 +1,27 @@
+package transformer
+
+import (
+	"github.com/Azure/sonic-mgmt-common/translib/tlerr"
+)
+
+// SaveStartupConfig initiates the operation of saving the current content of
+// the ConfigDB to a file that is then used to populate the database during the startup.
+func SaveStartupConfig() error {
+	r := HostQuery("cfg_mgmt.save", []string{})
+	if r.Err != nil {
+		return tlerr.New("internal SONiC Hostservice communication failure: %w", r.Err.Error())
```

Comment:
IMO `%w` should not be used here.. It is only recognized by `fmt.Errorf` and not by other Sprintf like functions.

Reply:
Done